src/components/SignupForm/SignupForm.tsx

```diff
@@ -259,87 +303,31 @@ const SignupForm = () => {
 
           <div className="!mb-6 !mt-2 flex flex-col gap-y-2">
             <div className="flex items-center gap-x-16 lg:gap-x-20">
-              <div className="flex w-28 items-center gap-x-2 lg:w-auto ">
-                <div>
-                  {errors.password?.message &&
-                  !/[a-z]+/.test(watch("password")) ? (
-                    <SmallAlertExclamation />
-                  ) : /[a-z]+/.test(watch("password")) ? (
-                    <SmallTick />
-                  ) : (
-                    <NeutralCircle />
-                  )}
-                </div>
-                <p
-                  className={cn("", {
-                    "text-red-500":
-                      errors.password && !/[a-z]+/.test(watch("password")),
-                  })}
-                >
-                  {pageContent.lowercase}
-                </p>
-              </div>
-
-              <div className="flex w-28 items-center gap-x-2 lg:w-auto">
-                <div>
-                  {errors.password && !/[0-9]+/.test(watch("password")) ? (
-                    <SmallAlertExclamation />
-                  ) : /[0-9]+/.test(watch("password")) ? (
-                    <SmallTick />
-                  ) : (
-                    <NeutralCircle />
-                  )}
-                </div>
-                <p
-                  className={cn("", {
-                    "text-red-500":
-                      errors.password && !/[0-9]+/.test(watch("password")),
-                  })}
-                >
-                  {pageContent.numbers}
-                </p>
-              </div>
+              <PassWordValidationIndicator
+                contentKey="lowercase"
+                passwordValidationRegEx={passwordValidationRegEx}
+              />
+              <PassWordValidationIndicator
+                contentKey="numbers"
+                passwordValidationRegEx={passwordValidationRegEx}
+              />
```

Comment:
In all component, same `passwordValidationRegEx` has been passed. Two approach can be taken here

1. Don't pass regEx as prop rather use `passwordValidationSchema` directly inside component OR
2. Pass only specific regEx